baseband/tests/test_conversion.py

```diff
@@ -177,7 +177,7 @@ def test_stream(self, tmpdir):
         vdif_file = str(tmpdir.join('converted.vdif'))
         # create and fill vdif file with converted data.
         with vdif.open(vdif_file, 'ws', nthread=data.shape[1],
-                       header=header) as fw:
+                       header=header, sample_rate=header.sample_rate) as fw:
```

Comment:
I don't think any changes where we get the sample rate from the header are necessary. (I do like the change in the documentation, since there you get it from the reader, and it helps suggest what one should do more generally).